server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/MessageAppender.java

```diff
@@ -176,35 +170,29 @@ private Flags getFlags(CreationMessage message) {
     }
 
     private ImmutableList<MessageAttachmentMetadata> getMessageAttachments(MailboxSession session, ImmutableList<Attachment> attachments) throws MailboxException {
```

Comment:
Should we remove `throws MailboxException`?
Because the catch this exception is already